docs/source/guide/pec-3-options.myst

```diff
@@ -415,3 +415,216 @@ uncertainty of the PEC result. For example, given the raw samples contained in
+whether the optimizer exited successfully, in addition to the optimized noise strength, which can then be input into 
+{func}`.representations.depolarizing.represent_operation_with_local_depolarizing_noise` to calculate quasiprobability representations of the
+ideal gate in terms of noisy gates.
+
+```{code-cell} ipython3
+:tags: ["skip-execution"]
+representations = represent_operation_with_local_depolarizing_noise(
+    operations_to_learn, epsilon_opt
+)
+```
+
+#### Learning quasiprobability representations with a biased noise model
+
+In cases where the noise of the backend can be approximated by a combined depolarizing and dephasing noise model with a bias factor, also
+referred to as a biased noise model, `pec.representations.learning.learn_biased_noise_parameters` can be used to learn the noise strength
+`epsilon` and noise bias `eta`  associated to a set of input operations. 
+
+
+The single-qubit biased noise channel is given by:
+
+$$
+\mathcal{D}(\epsilon) = (1 - \epsilon)[\mathbb{1}] 
++ \epsilon(\frac{\eta}{\eta + 1} \mathcal{Z}
++ \frac{1}{3}\frac{1}{\eta + 1}(\mathcal{X} + \mathcal{Y}
++ \mathcal{Z}))
+$$ 
+
+
+For multi-qubit operations, the noise channel used is the tensor product of the local single-qubit channels.
+
+
+The learning process is based on the execution of a set of training circuits on a noisy backend via a noisy
+{ref}`executor <guide/executors/executors>` and on a classical simulator via an ideal {ref}`executor <guide/executors/executors>`. 
+The training circuits are near-Clifford approximations of the input circuit. 
+During training, the noise strength and noise bias parameters are used to calculate quasiprobability representations of the ideal gate with
+a biased noise model.
+The representations are then input into {func}`.pec.execute_with_pec` to obtain an error-mitigated expecation value from execution of the
+training circuit, for comparison with the ideal expecation value obtained from classical simulation of the training circuit.
+The optimizer used in the learning function is from {py:func}`scipy.optimize.minimize`.
+The default optimization method in the learning function is `Nelder-Mead`, as that appears to work best for this particular problem setup.
+
+
+In addition to specifying the input operation, the circuit of interest, and the ideal and noisy executors,
+the user should specify the number of training circuits, the fraction of non-Clifford gates in the training circuits, and initial guesses for
+noise strength and noise bias. 
+The user can also set options for the intermediate executions of {func}`.pec.execute_with_pec` during the training process as a dictionary in
+`pec_kwargs`, specify {ref}`observable <guide/observables/observables>` of which the expecation value is to be computed, and
+enter a dictionary of additional data and options including optimization method (supported by {py:func}`scipy.optimize.minimize`) and settings
+for the chosen optimization method.
```

Comment:
I hesitate to change the structure as the two sub-sections are for different noise models. I would expect most users to consult them for different problems, instead of sequentially for the same problem.

Reply:
I moved much of the description and explanation as well as the note to the top, under the main section heading for learning-based PEC. That way, I was able to remove most of the redundant text.

Reply:
@natestemen WDYT of the new arrangement?

Unfortunately what I can see of the RTD build is using my old version 😢